### PR TITLE
added black btn

### PR DESCRIPTION
**Pull request by zahra-abdullah**
Issue:  https://github.com/shahednasser/sbuttons/issues/140

### CODE REVIEW

**Review by shahednasser — CHANGES_REQUESTED** [review]
Please compile the less files by running `npm run compile`. Currently, changes can't be seen

**Comment by zahra-abdullah** [comment]
> Please compile the less files by running `npm run compile`. Currently, changes can't be seen

Please see the compiled files added. Sorry about it.

File: src/components/_basic.less [hunk 1]
```diff
@@ -72,3 +72,12 @@ a.sbtn {
 .purple-btn {
   .colorRule(@purple);
 }
+
+.black-btn {
+  .colorRule(@black);
+
+  &:focus {
+    box-shadow: 0px 0px 0 0.2rem fade(@lightText, 30%);
```
Comment (shahednasser) [inline src/components/_basic.less:80]: Please don't use light text, use a new variable. Assign the variable the color `#7d7d7d` and fade percentage `67%`

**Review by shahednasser — CHANGES_REQUESTED** [review]

File: src/components/_basic.less (hunk 1 above)
Comment (zahra-abdullah) [reply src/components/_basic.less]: suggested change is added and branch is updated! please review! thank you!

**Comment by shahednasser** [comment]
Awesome! Thank you.

@all-contributors please add @zahra-abdullah for code

**Comment by zahra-abdullah** [comment]
> Awesome! Thank you.
> 
> @all-contributors please add @zahra-abdullah for code

thank you!

**Pull request merged**